CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(ENABLE_GOOGLETEST_INSTALL "Skip installation" OFF )
```

Comment:
Okay I see.  So should it SKIP install by default?  Or should I change it to 'DISABLE_GOOGLETEST_INSTALL and test if( NOT ) in the others?